docs/models/google.md

```diff
@@ -199,7 +199,25 @@ agent = Agent(model)
 
 ## Document, Image, Audio, and Video Input
 
-`GoogleModel` supports multi-modal input, including documents, images, audio, and video. See the [input documentation](../input.md) for details and examples.
+`GoogleModel` supports multi-modal input, including documents, images, audio, and video.
+
+YouTube video URLs can be passed directly to Google models:
```

Comment:
If we make this edit, I think we should add the Files API example from input.md "Uploaded Files" as well